shellcheck.1.md

```diff
@@ -56,6 +56,11 @@ not warn at all, as `ksh` supports decimals in arithmetic contexts.
     standard output. Subsequent **-f** options are ignored, see **FORMATS**
     below for more information.
 
+**-S**\ *SEVERITY*,\ **--severity=***severity*
+
+:   Specify maximum severity of errors to consider. Valid values are *error*,
```

Comment:
Good question!  Given that ```error``` is more severe, you're probably right.

I did it based on the definition of the ```Severity``` type:

```
data Severity = ErrorC | WarningC | InfoC | StyleC deriving (Show, Eq, Ord)
```

This is similar to how Unix log levels use higher numbers for more detailed messages.

Happy to call *minimum*, since that seems more logical when dealing with something called ```---severity```.

Want me to redo that?  I'm not entirely sure what happens to this pull request if I redo and force push.  I'll have to read about that.  :-)

Reply:
Make the change, commit it and push it. There will then be two commits in the PR. No big deal. Git and GitHub take care of it. When @koalaman merges, he has the option of taking it as is or squashing it down.

Reply:
Glad to be of service. We do that all the time at $work using GitHubEnterprise. Some of our PRs with reviews and updates have 15 commits in them. We usually squash to one when we merge. When you do that, the commit messages are all included in the squashed commit message as bullet points, which you can edit as yo see fit before confirming the squash and merge.